vendor/assets/javascripts/vue-resource.js.erb

```diff
@@ -0,0 +1 @@
+<%= evaluate(Vue.minify ? 'dist/vue-resource.min.js' : 'dist/vue-resource.js') %>
```

Comment:
1. why not use `require_asset` instead of `evaluate`?
2. how about DRYing it: `<%= require_asset Vue.dev_or_minified('vue-resource') %>`?